concourse/pipelines/destroy-cloudfoundry.yml

```diff
@@ -296,7 +296,8 @@ jobs:
                 cp cf-tfstate/cf.tfstate updated-cf-tfstate/cf.tfstate
                 cd paas-cf/terraform/cloudfoundry || exit
                 terraform init
-                terraform destroy -force -var env="((deploy_env))" \
+                terraform -chdir=../../../paas-cf/terraform/cloudfoundry destroy \
+                  -auto-approve -var env="((deploy_env))" \
                   -var-file="../../../paas-cf/terraform/((aws_account)).tfvars" \
                   -var-file="../../../paas-cf/terraform/cloudfoundry/((aws_account)).tfvars" \
                   -var-file="../../../paas-cf/terraform/((aws_region)).tfvars" \
```

Comment:
And I think you may need to remove the trailing `paas-cf/terraform/cloudfoundry` on the line following this one :point_down: . Due to the `-chdir` this means it's now looking in `paas-cf/terraform/cloudfoundry/paas-cf/terraform/cloudfoundry` for the plan/state files..?

Reply:
🤦 
yes and yes. Thank you